packages/server/__tests__/api/grants.test.js

```diff
@@ -41,6 +41,19 @@ describe('`/api/grants` endpoint', () => {
         },
     };
 
+    const assignedBy = {
```

Comment:
Presumably this is pulled out here since it's used in both the tests you've added. I'd love to see it moved a bit closer to the code that uses it, like just within the closest common `context` block. 

Another angle to consider (warning: personal opinion, feel free to disregard if you disagree!)... I actually find denormalizing this and just writing it out in each test is often easier to read and work with. If it's repeated a ton of times, then yeah, I'd pull it out, but if it's incidentally the same across two tests, I actually lean slightly toward just writing it out both times inline. There's a big benefit to keeping tests simple and easy to read.

Reply:
Done here.